src/etwlogger_driver.c

```diff
@@ -260,16 +289,28 @@ void etwlogger_log(LOG_CATEGORY log_category, const char* file, const char* func
     {
```

Comment:
Could cut down a bunch of lines if we just set a `const char* text_to_log` to either `text` or `"INTERNAL LOGGING ERROR: failed in vprintf_alloc"`.

Then we don't need 2 switch statements

Reply:
Done.

---
In reply to: [425324491](https://github.com/Azure/azure-c-logging/pull/3#discussion_r425324491) [](ancestors = 425324491)